<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arinc665Qt::ImportMediaSetXmlSettingsPage</class>
 <widget class="QWizardPage" name="Arinc665Qt::ImportMediaSetXmlSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="xmlFileLabel">
     <property name="text">
      <string>XML File</string>
     </property>
     <property name="buddy">
      <cstring>xmlFile</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="xmlFileWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="xmlFile"/>
      </item>
      <item>
       <widget class="QPushButton" name="selectXmlFile">
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="inputDirecoryLabel">
     <property name="text">
      <string>Input Directory</string>
     </property>
     <property name="buddy">
      <cstring>inputDirectory</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="inputDirectoryWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="inputDirectory"/>
      </item>
      <item>
       <widget class="QPushButton" name="selectInputDirectory">
        <property name="icon">
         <iconset theme="document-open"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="arinc665VersionLabel">
     <property name="text">
      <string>ARINC 665 Version</string>
     </property>
     <property name="buddy">
      <cstring>arinc665Version</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="batchFileCreationLabel">
     <property name="text">
      <string>Batch File Creation</string>
     </property>
     <property name="buddy">
      <cstring>batchFileCreation</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="loadHeaderCreationLabel">
     <property name="text">
      <string>Load Header Creation</string>
     </property>
     <property name="buddy">
      <cstring>loadHeaderCreation</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="arinc665Version"/>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="loadHeaderCreation"/>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="batchFileCreation"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>xmlFile</tabstop>
  <tabstop>selectXmlFile</tabstop>
  <tabstop>inputDirectory</tabstop>
  <tabstop>selectInputDirectory</tabstop>
  <tabstop>arinc665Version</tabstop>
  <tabstop>loadHeaderCreation</tabstop>
  <tabstop>batchFileCreation</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
